### ISSUE TITLE
Examples and Documentation

### ISSUE THREAD

**Issue by GeorgeS2019** [DotnetNative/JNI#1]
Examples and Documentation

**Comment by 3fbaea00** [comment]
Yeah, it will has much examples and a little in-code documentation. But now the project is at the development stage, and t's not clear what it will look like in the final form. I can help you to understand, if you need contact me - discord: nativeaot, telegram: yotic
